bdsf/gausfit.py

```diff
@@ -645,7 +645,7 @@ def inigaus_nobeam(self, isl, thr, beam, img):
             x, y = N.where(~isl.mask_active); xcen = N.mean(x); ycen = N.mean(y)
             invm = ~isl.mask_active
             #bound = invm - nd.grey_erosion(invm, footprint = N.ones((3,3), int)) # better to use bound for ellipse fitting
-            mom = func.momanalmask_gaus(invm, N.zeros(invm.shape, dtype=N.int16), 0, 1.0, True)
+            mom = func.momanalmask_gaus(invm, N.zeros(invm.shape, dtype=int), 0, 1.0, True)
```

Comment:
Here you quadruple the size of the array of zeros, because `dtype=int` is an alias for `dtype=int64`.

Reply:
Indeed, but the array is small in practice (as above).